app/controllers/storage_locations_controller.rb

```diff
@@ -79,6 +79,9 @@ def import_inventory
       flash[:notice] = "Inventory imported successfully!"
       redirect_back(fallback_location: storage_locations_path(organization_id: current_organization))
     end
+  rescue Errors::InventoryAlreadyHasItems => e
```

Comment:
This shouldn't be a thing. In general they _will_ have items and that's not an error.

Reply:
Ok, i hadn't understood that!  Note that on the frontend, the button to import inventory only appears if there is nothing in the current inventory

```
<%= modal_button_to("#csvImportModal", { text: "Import Baseline Inventory", icon: "upload" }) if @storage_location.items.empty? %>
```
Which is why i got confused, to me the ticket was about not making it possible to import while there was items. We should have this button visible at all times then no?